<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choiceLoginDevWidget</class>
 <widget class="QWidget" name="choiceLoginDevWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#choiceLoginDevWidget { 
border-image: url(:/monres/background.bmp); 
} 
#choiceLoginDevWidget * { 
border-image:url(); 
}</string>
  </property>
  <widget class="QPushButton" name="pvmsPushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>274</width>
     <height>97</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-image: url(:/monres/preview.bmp);}
QPushButton:hover{background-color: rgb(246, 246, 246);color: rgb(86, 164, 246);border-color: rgb(255, 255, 255);border-width: 2px;border-style: solid;}

QPushButton:pressed
{background-color: rgb(86, 164, 246);color: rgb(255, 255, 255);border-color: rgb(170, 170, 170);border-width: 2px;border-style: solid;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cvmsPushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>274</width>
     <height>97</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-image: url(:/monres/playback.bmp);}

QPushButton:hover{background-color: rgb(246, 246, 246);color: rgb(86, 164, 246);border-color: rgb(255, 255, 255);border-width: 2px;border-style: solid;}

QPushButton:pressed
{background-color: rgb(86, 164, 246);color: rgb(255, 255, 255);border-color: rgb(170, 170, 170);border-width: 2px;border-style: solid;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="versionLabel">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>680</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>V1.0.0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
